### ISSUE TITLE
Drone configuration fails with "Check these paths: drone_token" while using a self-hosted instance of shields

### ISSUE THREAD

**Issue by Bhargav-Rao** [badges/shields#4685]
Are you experiencing an issue with...

- [ ] [shields.io](https://shields.io/#/)
- [x] My own instance
- [ ] [gh-badges NPM package](https://www.npmjs.com/package/gh-badges)

:beetle: **Description**

I have been trying to set up a self hosted shields, with my own GH Enterprise server and my own self hosted Drone CICD server. It seems to be working fine until I add `drone_token` as one of the configuration parameters. I get the following error: 

```
Configuration:
{ bind: { address: '0.0.0.0' },
  metrics: { prometheus: { enabled: false } },
  ssl: { isSecure: false },
  cors: { allowedOrigin: [] },
  persistence: { dir: './private' },
  services:
   { github:
      { baseUri: 'https://github.bhargavrao.com/api/v3/',
        debug: { enabled: false, intervalSeconds: 200 } },
     trace: false },
  profiling: { makeBadge: false },
  cacheHeaders: { defaultCacheLengthSeconds: 120 },
  rateLimit: true,
  handleInternalErrors: true,
  fetchLimit: '10MB' }
/usr/src/app/core/server/server.js:139
      throw Error(
      ^
Error: Private configuration is invalid. Check these paths: drone_token
    at new Server (/usr/src/app/core/server/server.js:139:13)
    at Object.<anonymous> (/usr/src/app/server.js:49:34)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
```

:link: **Link to the badge**

:bulb: **Possible Solution**

The error seems to be coming from [Line 90 in server.js](https://github.com/badges/shields/blob/master/core/server/server.js#L90), as there is no `drone_token` in that list. I am not really sure that this is the problem, though. 

Thanks for your help, and the shields product. I've been really liking it.

**Comment by calebcartwright** [comment]
Hey @Bhargav-Rao 👋 How are you setting the value for your Drone token? via an env var or adding it to the config file?

**Comment by paulmelnikow** [comment]
Either way, it seems like that schema is missing drone_token.

**Comment by calebcartwright** [comment]
> Either way, it seems like that schema is missing drone_token.

Ahh so it is. TeamCity missing from this one too. Let me see if I can open a PR to fix this

**Comment by Bhargav-Rao** [comment]
Hey @calebcartwright, I'm setting it through an environment variable.

**Comment by chris48s** [comment]
closed in  #4689
